docs/Course Wiki/PHYS Course Offerings/PHYS213.md

```diff
@@ -0,0 +1,56 @@
+# PHYS213
+
+PHYS213 (Thermal Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
+
+## Content Covered
+
+- Equilibrium and entropy
+- Heat and temperature
+- Ideal gases
+- Heat capacity
+- Thermodynamic/Reversible processes
+- Boltzmann factor
+- Helmholtz/Gibbs free energy
+  
+PHYS213 is the introduction to thermodynamics for ECE majors. Being only 8 weeks long, the course does not go too in depth about most topics, but most ECE majors do not need to know more than the cursory knowledge provided.
+
+## Prerequisites
+
+- [PHYS211](PHYS211.md)
+- [MATH241](../MATH%20Course%20Offerings/MATH241.md) (Corequisite)
+
+The official prerequisites for PHYS213 are [PHYS211](PHYS211.md) and credit or concurrent registration in [MATH241](../MATH%20Course%20Offerings/MATH241.md). There is almost no direct overlap in content between the physics courses, and very little calculus, in general, is used.
+
+## When to Take It
+
+ECE majors typically take PHYS213 and [PHYS214](PHYS214.md) during the same semester after taking [PHYS212](PHYS212.md). The two courses are designed to be taken in the same semester, so it ends up filling roughly the same amount of time as a normal 4 credit hour, with the content changing halfway through.
+
+## Course Structure
+
+PHYS213 holds 2 lectures per week, with prelectures and checkpoints to complete before each lecture. The prelectures and checkpoints are where students will do most of the learning, with the lectures being mainly for reexplaination and clarification. 
+
+Homework is assigned after each lecture in smartPhysics (free for PHYS213). The homework for each week's lectures are typically due at the same time early during the following week.
+
+There are 2 midterm exams and 1 final exam, all taken through Prarielearn at the CBTF. 
+
+There are 3 bi-weekly labs.
+
+## Instructors
+
+In the past, this course has been taught by Professors Wagner and Covey.
+
+## Course Tips
+
+The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study.
+
+## Life After
+
+Graduation
+
+## Infamous Topics
+
+- Entropy: It is a very strange concept, so watching the prelecture a few times is recommended.
+
+## Resources
+
+Course website
```

Comment:
Link website

Reply:
The website changes every semester...

Reply:
Sorry, remove course website for both then